src/web/stylesheets/themes/_solarizedDark.css

```diff
@@ -138,6 +138,7 @@
 
     /* Misc. */
     --drop-file-border-colour: var(--base01);
+    --table-border-colour: rgba(131, 148, 150, 0.4);
```

Comment:
Please could you use the predefined Solarized colour variables instead of specific RGBA values. I think `base01` should work nicely here